### DISCUSSION TITLE
openfass-express-typescript using cotalker library: error econnrefused

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#100] [2. Questions and Answers] [answered]
Hola!

Estoy usando opefaas-express-typescript, y cuando uso alguna funcion de cotalker library, me aparece el error de econnrefused ! 
Deseo hacer las llamadas a mi local de k3d, pero no sé si tengo que agregar o cambiar algo en la libreria para que me funcione la llamada de red. (Intente hacer una llamada de red al local desde postman, y me funciona bien)
Si alguien me pueda ayudar con este tema, sería genial! gracias!

Error:
``` 
error [2021-04-20T20:36:18.105Z] ERROR: 18837 Error: connect ECONNREFUSED 127.0.0.1:9000 
nyi [2021-04-20T20:36:18.105Z] NYI : reportError 
Error: connect ECONNREFUSED 127.0.0.1:9000
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1129:14) {
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000
}
```

**Comment by guillevalin** [comment, marked as answer]
@ValeMP estás tratando de acceder a la api de cotalker a través de localhost:9000 o 127.0.0.1:9000 la nueva URL de la API en K3D es http://cotalker

**Reply by ValeMP** [reply]
segun @acedward debería funcionar, pero he intentado cambiar el basURL y no sé como dejar de que apunte al puerto 9000

**Reply by guillevalin** [reply]
@ValeMP en el template que me mencionas, hay una llamada a core de ejemplo que tiene la siguiente URL:
`return await requestpromise('https://cotalker.com/api/v1/users/me', { headers, json: true });`

En el caso del desarrollo en K3D, esa URL hay que reemplazar `https://cotalker.com` por `http://cotalker`

**Reply by acedward** [reply]
No hay feedback. Voy a cerrarlo.
